opencog/attentionbank/bank/AttentionBank.cc

```diff
@@ -237,7 +237,8 @@ double AttentionBank::getNormalisedZeroToOneSTI(AttentionValuePtr av,
 AttentionBank& opencog::attentionbank(AtomSpace* asp)
 {
     static AttentionBank* _instance = nullptr;
-    static AtomSpace* _as = nullptr;
+    static AtomSpace* _as;
+    _as->get_uuid();
```

Comment:
For the first `opencog::attentionbank(AtomSpace* asp)` call the `_as` pointer is initialiazed in some garbage  `_as->get_uuid()` has undefined behavior.

The idea of the fix was to compare `asp` and `_as` by their uuid if they both are not null.